src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java

```diff
@@ -40,8 +40,8 @@ private AnomalyDetectorSettings() {}
 
     public static final Setting<Integer> MAX_SINGLE_ENTITY_ANOMALY_DETECTORS = Setting
         .intSetting(
-            "opendistro.anomaly_detection.max_anomaly_detectors",
-            1000,
+            "opensearch.anomaly_detection.max_anomaly_detectors",
```

Comment:
I know @saratvemulapalli has been saying that we would want these to be `plugins.` just to avoid renaming again :) Care to comment?

Reply:
I think it was miss on our side, we haven't updated the guidance. I would prefer to have `plugins` instead of `opensearch`.
Can we update the doc: https://github.com/opensearch-project/opensearch-plugins/blob/main/CONVENTIONS.md#settings

Also let folks working on JobScheduler to take care of it as well.

Reply:
Do you mean we should change "opensearch.anomaly_detection.xxx" to "plugins.anomaly_detection.xxx"?

Sure, I will make that change.
Thank you for the comments!

Reply:
Yes please. Thanks Alex!

Reply:
And why would we not make this `anomaly_detection.max_anomaly_detectors`? That's what k-nn is doing, for example.

Reply:
Definitely that works too. But I would love to have consistency across different pieces in the plugins.
And, we have decided to go with that as we have already discussed in the public:
https://github.com/opensearch-project/opensearch-plugins/issues/13
https://discuss.opendistrocommunity.dev/t/thoughts-on-having-opensearch-identifier-in-all-things-plugins/5855

I am not opposed to either way but since we have taken a decision in the public, I would prefer to stick with it unless we see strong contentions to why that doesnt work. What do you think?

Reply:
A-OK by me to include `plugins`. Who's updating the doc? :)

Reply:
Haha oncall :/, thanks for the change!

Reply:
Thank you very much for the input!
I have submitted a commits to update "opensearch" to "plugins"